src/presentation/cli/CollectMetrics.ts

```diff
@@ -0,0 +1,48 @@
+import { promises as fs } from 'fs'
+import {
+  createPublicClient,
+  http
+} from 'viem'
+import { hemiSepolia, hemi } from 'hemi-viem'
+
+const chain = process.env['TESTNET'] ? hemiSepolia : hemi
+const filePath = process.env['METRICS_FILE_PATH'] ?? ''
+
+const client = createPublicClient({ chain, transport: http() })
+const blockExplorerUrl = chain.blockExplorers.default.url
+
+const execute = async (): Promise<void> => {
+  console.info('Hemi Connector | Collecting chain metrics...')
+
+  try {
+    const [blockNumber, stats] = await Promise.all([
+      client.getBlockNumber(),
+      fetch(
+        `${blockExplorerUrl}/api/v2/stats`)
+        // eslint-disable-next-line @typescript-eslint/promise-function-async
+        .then(response => response.json())
+    ])
+
+    const totalTransactions = parseInt(stats.total_transactions)
+    const content = JSON.stringify({
+      'btc-transactions': totalTransactions.toString(),
+      'latest-keystone': blockNumber.toString()
+    })
```

Comment:
Let's add a `timestamp` property to the JSON file so we can easily check from the consumer side if the contents are fresh or not.

Reply:
added